resolwe/flow/executors/startup_processing_container.py

```diff
@@ -500,7 +506,9 @@ def send_chunk(processing_filenames):
             )
             if self.upload_socket.recv(1) == b"0":
                 raise RuntimeError(
-                    "Error sending filenames: {}.".format(processing_filenames)
+                    "Error sending filenames: {} (comminication container response indicates error).".format(
```

Comment:
Would it make sense to move the comment (in parenthesis) before the colon, just if the list of filenames is long?

Reply:
I have rephrased the error message and split it into two rows.